server/gen_video.py

```diff
@@ -21,8 +21,14 @@
 import random
 import numpy as np
 import cv2
+from subprocess import call, PIPE
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+if "test_videos" not in dir_path:
+	dir_path += "/test_videos/"
+
+call(f"mkdir {dir_path}".split())
```

Comment:
Consider adding logs to make the behavior more transparent. Same for the other steps below.